# 第1章 物质的基本性质

🔖为保持章节序号与课本一致，此处为章标题占位符，具体内容请参阅对应章的 ipynb 文件。

# 第2章 流体力学基础

In [3]:
import sympy as sp

sp.init_session()

IPython console for SymPy 1.14.0 (Python 3.13.9-64-bit) (ground types: python)

These commands were executed:
>>> from sympy import *
>>> x, y, z, t = symbols('x y z t')
>>> k, m, n = symbols('k m n', integer=True)
>>> f, g, h = symbols('f g h', cls=Function)
>>> init_printing()

Documentation can be found at https://docs.sympy.org/1.14.0/



## 流体力学简介



## 理想流体的定常流动

根据质量守恒定律，任取定常流动的一细流管，经过时间$\Delta t$，流入的流体质量 $\Delta m_1 = \rho_1S_1v_1\Delta t$ 与流出的质量 $\Delta m_2 = \rho_2S_2v_2\Delta t$ 相同

$$ \rho_1S_1v_1\Delta t = \rho_2S_2v_2\Delta t\tag{2.1a}$$
或
$$ \rho Sv=常量 \tag{2.1b}$$

该细流管任意垂直截面均满足式$(2.1\text{b})$

如果假设流体式不可压缩的，则它的密度不变，式$(2.1\text{b})$可以简化为
$$ Q=Sv=常量 \tag{2.2}$$

式中 $Q$ 为流管的流量。

上式称为流体的连续性原理。

## 伯努利方程及其应用

### 伯努利方程的推导

在做定常流动的理想流体中任取一根细流管，

$$ p_1 + \frac{1}{2}\rho v_1^2 + \rho g h_1 = p_2 + \frac{1}{2}\rho v_2^2 + \rho g h_2 \tag{2.3a}$$
或
$$ \rho + \frac{1}{2} \rho v^2 + \rho g h=常量 \tag{2.3b}$$
式$(2.3)a$或$(2.3b)$即伯努利方程


伯努利方程表明，在同一细流管的不同截面处，单位体积流体的动能 $\frac{1}{2}pv^2$、单位体积流体的势能$\rho gh$、单位体积流体的压强能$p$三项之和为常量。

在工程上，伯努利方程常写成：
$$ \frac{p}{\rho g} + \frac{v^2}{2g} + h=常量 \tag{2.4} $$
上式左边三项依次称为压强头、速度头和高度头。

## 黏性流体的定常流动
### 牛顿黏性定律

实验证明，实际流体在固体管中流动时，如果流速不是很大，流体将做分层流动，此时不同流层得流速不同，但流层之间只做相对滑动彼此不相混合，这种流动状态称为层流（laminar flow）。

若取层间距 $\Delta y \to 0$，$\Delta v$ 与 $\Delta y$ 之比得极限值就反映了流速得空间变化率，称为**速度梯度**，其定义为
$$ \lim_{\Delta y\to 0}\frac{\Delta v}{\Delta y}=\frac{\operatorname{d}v}{\operatorname{d}y}  $$

实验证明，相邻流层之间的摩擦系数 $F_f$ 的大小，与两流层的接触面积 $S$ 成正比，与速度梯度 $\frac{\operatorname{d}v}{\operatorname{d}y}$ 成正比，即有
$$ F_f=\eta \frac{\operatorname{d}v}{\operatorname{d}y}S \tag{2.10}$$

上式中的比例系数 $\eta$ 称为流体的黏度（viscosity），它的单位是 $\mathrm{Pa\cdot s}$，该公式称为**牛顿黏性定律**。

### 黏性流体的伯努利方程

若单位体积的黏性流体从 1 到 2 时摩擦损耗的机械能用 $\Delta w_{21}$ 表示，应用功能原理，就可以得到黏性流体的伯努利方程为
$$ p_1 + \frac{1}{2}\rho v_1^2 + \rho g h_1 = p_2 + \frac{1}{2}\rho v_2^2 + \rho g h_2 + \Delta w_{21}\tag{2.11}$$



### 湍流 雷诺数

实验表明，当流体的流速增大到一定数值后，层流将被破坏，相邻层流间不但有滑动，还有混合。这是流体做不规则运动，有垂直于流管轴线方向的分速度产生，这种运动称为湍流（turbulent flow）。流体做湍流时，阻力大流量小，能量损耗增加。实验证明，能量损耗 $\Delta E$ 与速度的关系为
$$ \Delta E=kv^2 \tag{2.12} $$
式中 $k$ 是比例常数，它与管道的形状、大小以及管道的材料有关，$v$ 是平均流速。

流体的流动状态由雷诺数 $\rho vl/\eta$ 决定，雷诺数小时做层流，雷诺数大时做湍流。流体由层流到湍流过渡的雷诺数，叫临界雷诺数（critical Reynolds number），记作 $Re$。对于圆形管道，引入
$$ Re=\frac{\rho v d}{\eta} \tag{2.13}$$

## 泊肃叶定律 斯托克斯定律

### 泊肃叶定律

黏性流体在粗细均匀的管道中做稳定流动时，流量与压强、黏度、管的粗细和长短等因素的关系。

圆管中实际流体的流速随半径的分布规律:

$$v=\frac{p_1-p_2}{4\eta l}\big(R^2-r^2\big) \tag{2.14}$$

泊肃叶方程：

$$ Q=\frac{\pi R^4}{8\eta l}(p_1-p_2)=\frac{\pi R^4}{8\eta l}\Delta p \tag{2.15}$$

达西定律：
若将 $R'=\frac{8\eta l}{\pi R^4}$ 代入式$(2.15)$，得
$$Q=\frac{p_1-p_2}{R'} \tag{2.16}$$

此式与电学中的欧姆定律相似，因此把 $R'$ 称为流体阻力系数(简称流阻)。式$(2.16)$是生命科学中常用的公式，称为达西定律(Darcy theorem)。



### 奥氏黏度计

$$\frac{\eta_2}{\eta_1}=\frac{\rho_2\, t_2}{\rho_1\, t_1}$$



## 思考题
### 思考题2.1
伯努利方程成立的条件是什么？泊肃叶方程成立的条件又是什么？

🏍️解答：
伯努利方程成立条件：理想流体、定常流动、同一流管。

泊肃叶方程成立条件：黏性流体、定常层流、水平圆管。

### 思考题2.2
如何解释“巷小风大”？

🏍️解答：

巷道狭窄，空气流通截面积小，根据连续性方程 $S_1v_1=S_2v_2$，截面积变小流速增大；对应伯努利原理，流速大处压强低。

### 思考题2.3

如何解释“河道宽处水流缓，河道窄处水流急”？

🏍️解答：

河水近似不可压缩流体，连续性方程 $S v=\text{常量}$。河道宽处横截面积$S$大，则流速$v$小；河道窄处横截面积$S$小，则流速$v$大。

### 思考题2.4

定常流动是理想流体所特有的吗？黏性流体有无定常流动？

🏍️解答：

不是理想流体特有。黏性流体也可以发生定常流动：空间各点流速不随时间变化，只是层内存在内摩擦力，泊肃叶公式描述的圆管层流就是黏性流体的定常流动。

### 思考题2.5

平行行驶的两艘船为什么要保持一定距离？

🏍️解答：

两船内侧水流通道变窄，水流流速大，压强小；船外侧水流流速小、压强大。内外压强差会把两船相互推向一起，容易发生碰撞，因此必须保持间距。

### 思考题2.6

液体在管中流动时，从连续性原理来看，管子越粗，流速越小，而从泊肃叶方程来看，管子越粗，流量越大，这两者是否矛盾，为什么？

🏍️解答：

不矛盾。

连续性方程 $\(Sv=Q\)$：流量$Q$固定不变前提下，截面积$S$越大流速$v$越小。

泊肃叶公式 $Q=\dfrac{\pi R^4\Delta p}{8\eta l}$：管道两端压强差$\Delta p$固定不变前提下，半径越大流量$Q$越大。

两个公式的前提条件不一样。

### 思考题2.7

水平的玻璃管，由截面均匀而截面积不等的A、B、C三段串联而成。水从A段进入，从C段流出。在三段管壁上各打一小孔，A段上小孔有水射出，B段小孔有气泡出现，C段小孔不射水，也无气泡，设水为理想流体。问三段管子哪段内径最大，哪段内径最小，为什么？

🏍️解答：

- B段内径最小：流速最大，压强低于大气压，所以空气被压入形成气泡。
- A段内径最大：流速小，压强大于大气压，小孔向外喷水。
- C段压强等于大气压，流速介于两者之间。


### 思考题2.8

生物流体同一般流体有何区别？

🏍️解答：

1. 很多生物流体（血液）属于非牛顿流体，黏度会随流速/切变率改变，不服从牛顿黏性定律；
2. 生物流体成分复杂，含有细胞、大分子颗粒；
3. 生物管道是弹性可形变管道，不是刚性管壁；
4. 流动多为脉动非定常流动。

## 习题

### 习题2.1

在如图所示的装置中，液体在水平管道中流动，截面B与大气相通。试求盆中液体能够被吸上时，$h$的表达式(设$S_A$、$S_B$分别为管道A、B处的截面积，$Q$为秒流量，$C$点与大气相通，$p_C=p_0$)。

🏍️解答：

取B点、C点，$p_C=p_0$，B与大气相通$p_B=p_0$。

连续性方程：$Q=S_A v_A=S_B v_B$；

对A、B伯努利（水平）：

$$p_A+\frac12\rho v_A^2=p_0+\frac12\rho v_B^2$$

盆中液体被吸上，$p_A+\rho g h = p_0$

联立解得：

$$h=\dfrac{Q^2}{2g}\left(\dfrac{1}{S_A^2}-\dfrac{1}{S_B^2}\right)$$



### 习题2.2

变截面管宽部分的截面积$S_1=0.08\ \mathrm{cm^2}$，窄部分的截面积$S_2=0.04\ \mathrm{cm^2}$，两处间的压强差为25 Pa，求管中宽部流体流动的速度。已知液体密度为$1059.5\ \mathrm{kg\cdot m^{-3}}$。

🏍️解答：

$S_1=0.08\ \mathrm{cm^2},\ S_2=0.04\ \mathrm{cm^2}$，$\Delta p=25\ \mathrm{Pa}$，$\rho=1059.5\ \mathrm{kg/m^3}$

连续性：$S_1 v_1=S_2 v_2 \Rightarrow v_2=2v_1$

伯努利水平管：

$$p_1+\frac12\rho v_1^2 = p_2+\frac12\rho v_2^2$$
$$\Delta p=\frac12\rho(v_2^2-v_1^2)=\frac12\rho(4v_1^2-v_1^2)=\frac32\rho v_1^2$$
$$v_1=\sqrt{\dfrac{2\Delta p}{3\rho}}=\sqrt{\dfrac{2\times 25}{3\times 1059.5}}\approx \boldsymbol{0.126\ \mathrm{m/s}}$$

### 习题2.3

如图所示，水平管的横截面积在粗处为$40\ \mathrm{cm^2}$，在细处为$10\ \mathrm{cm^2}$，水的流量为$3\times10^{-3}\ \mathrm{m^3\cdot s^{-1}}$。水平管下面装有一个U形管，管内下部盛有水银，已知水银密度为$13.6\times10^3\ \mathrm{kg\cdot m^{-3}}$。求：(1)水在粗处和细处的流速；(2)两处的压强差；(3)U形管中水银柱的高度差。

🏍️解答：

已知：$S_1=40\ \mathrm{cm^2}=40\times10^{-4}\mathrm{m^2}$，$S_2=10\ \mathrm{cm^2}=10\times10^{-4}\mathrm{m^2}$，$Q=3\times10^{-3}\mathrm{m^3/s}$，$\rho_\text{Hg}=13.6\times10^3\mathrm{kg/m^3}$
1. 流速

$$v_1=\frac{Q}{S_1}=\frac{3\times10^{-3}}{40\times10^{-4}}=0.75\ \mathrm{m/s}$$
$$v_2=\frac{Q}{S_2}=\frac{3\times10^{-3}}{10\times10^{-4}}=3.0\ \mathrm{m/s}$$

2. 压强差（水平伯努利）
$$\Delta p=p_1-p_2=\frac12\rho(v_2^2-v_1^2)=0.5\times1000\times(3^2-0.75^2)=4218.75\ \mathrm{Pa}$$

3. U型管水银高差：$\Delta p=(\rho_\text{Hg}-\rho_\text{水}) g h$
$$h=\frac{\Delta p}{(\rho_\text{Hg}-\rho_\text{水})g}=\frac{4218.75}{(13600-1000)\times9.8}\approx 0.0342\ \mathrm{m}=3.42\ \mathrm{cm}$$

### 习题2.4

如图所示，水定常地流过直径为0.1 m的管子，经喷管射入大气，喷管直径为0.05 m。如果截面$b$上的水速为$20\ \mathrm{m\cdot s^{-1}}$，求截面$a$上流体的压强。

🏍️解答：

$d_a=0.1\mathrm{m},\ d_b=0.05\mathrm{m},\ v_b=20\mathrm{m/s}$；出口$b$通大气$p_b=p_0$。
连续性 $S_a v_a=S_b v_b$，$S\propto d^2$
$$v_a=v_b \cdot \frac{d_b^2}{d_a^2}=20\times \frac{0.05^2}{0.1^2}=5.0\ \mathrm{m/s}$$
$a$比$b$高$4\mathrm{m}$，伯努利：
$$p_a+\frac12\rho v_a^2+\rho g h_a = p_b+\frac12\rho v_b^2+\rho g h_b$$
$h_a-h_b=4\mathrm{m}$
$$
\begin{align*}
p_a &= p_0 +\frac12\rho(v_b^2-v_a^2)+\rho g(h_b-h_a) \\
&=1.013\times10^5 + 0.5\times1000(400-25)+1000\times9.8\times(-4) \\
&=2.70\times 10^5\ \mathrm{Pa}
\end{align*}
$$


### 习题2.5

半径为0.02 m的水管以$0.01\ \mathrm{m^3\cdot s^{-1}}$的流量输送水，水温为$20^\circ \mathrm{C}$。问：(1)水的平均流速是多少？(2)流动是层流还是湍流？(3)要确定管中流体流动的最大速度，这些数据是否足够？

🏍️解答：


$R=0.02\mathrm{m},\ Q=0.01\mathrm{m^3/s}$；$20^\circ\text{C}$水$\eta=1.005\times10^{-3}\mathrm{Pa\cdot s}$

1\. 平均流速

$$S=\pi R^2=\pi\times0.02^2\approx 1.2566\times10^{-3}\ \mathrm{m^2}$$
$$\bar v=\frac{Q}{S}=\frac{0.01}{1.2566\times10^{-3}}\approx \boldsymbol{7.96\ \mathrm{m/s}}$$

2\. 雷诺数 $Re=\dfrac{\rho \bar v d}{\eta},\ d=2R=0.04\mathrm{m}$
$$Re=\frac{1000\times7.96\times0.04}{1.005\times10^{-3}}\approx 3.17\times10^5>2000$$
流动是湍流。

3\. 不足够。湍流管内流速分布不再是抛物线，仅知道流量求不出最大流速。

### 习题2.6

考虑理想不可压缩流体在收缩管道中的一维定常流动，如果管道截面积沿流动方向($x$方向)的变化规律为$(A(x))$，如图所示，由测量得知收缩管进口截面积为$A_0$，进口流速为$v_0$，试求收缩管中任意截面处的压强梯度$\dfrac{\mathrm{d}p}{\mathrm{d}x}$的表达式。设流体密度为$\rho$，管道水平放置。

🏍️解答：

一维定常不可压缩：$A(x)v(x)=A_0 v_0 \Rightarrow v(x)=\dfrac{A_0 v_0}{A(x)}$
水平伯努利：
$$p+\frac12\rho v^2=\text{常数}$$
两边对$x$求导
$$\frac{\mathrm{d}p}{\mathrm{d}x}+\rho v \frac{\mathrm{d}v}{\mathrm{d}x}=0$$
$$\frac{\mathrm{d}v}{\mathrm{d}x}=A_0 v_0 \cdot \left(-\frac{1}{A^2}\right)\frac{\mathrm{d}A}{\mathrm{d}x}$$
代入
$$\boldsymbol{\frac{\mathrm{d}p}{\mathrm{d}x}=\rho \frac{A_0^2 v_0^2}{A^3(x)} \frac{\mathrm{d}A(x)}{\mathrm{d}x}}$$



### 习题2.7

由于飞机机翼形状的关系，在机翼上面、下面间就形成压强差，因而产生使机翼上升的力。假定空气流过机翼是定常流动，并假定空气的密度不变，等于$1.29\times10^{-3}\ \mathrm{g\cdot cm^{-3}}$。如果机翼下面的气流速率为$100\ \mathrm{m\cdot s^{-1}}$，问要得到1 000 Pa的压强差，机翼上面的气流速率应该是多少？

🏍️解答：

$\rho=1.29\ \mathrm{kg/m^3}$，$v_\text{下}=100\mathrm{m/s}$，$\Delta p=p_\text{下}-p_\text{上}=1000\mathrm{Pa}$
伯努利：
$$p_\text{下}+\frac12\rho v_\text{下}^2 = p_\text{上}+\frac12\rho v_\text{上}^2$$
$$\Delta p=\frac12\rho\left(v_\text{上}^2-v_\text{下}^2\right)$$
$$v_\text{上}=\sqrt{v_\text{下}^2+\frac{2\Delta p}{\rho}}=\sqrt{100^2+\frac{2\times1000}{1.29}} \approx \boldsymbol{107.4\ \mathrm{m/s}}$$



### 习题2.8

如图所示，风平行地吹过一栋密封的房子，当风速为多大时房子内外的压强差为10%？假如房子密封得足够好，这一压强差会使得房子爆裂。已知空气密度为$1.293\ \mathrm{kg\cdot m^{-3}}$。

🏍️解答：

室内静止$v_\text{内}=0$；室外风速$v$；内外压强差$10\% p_0$；$p_0=1.013\times10^5\mathrm{Pa}$，$\Delta p=0.10\,p_0$
伯努利：$p_\text{内}=p_\text{外}+\dfrac12\rho v^2$
$$\Delta p=\frac12\rho v^2$$
$$v=\sqrt{\frac{2\times0.1\times 1.013\times 10^5}{1.293}} \approx \boldsymbol{125.2\ \mathrm{m/s}}$$



### 习题2.9

主管与细管间的压强差为$10^6\ \mathrm{Pa}$，主管和细管的横截面积分别为$0.2\ \mathrm{m^2}$和$0.1\ \mathrm{m^2}$，问管子中水的流量是多少？

🏍️解答：

$p_1-p_2=10^6\mathrm{Pa}$，$S_1=0.2\mathrm{m^2},\ S_2=0.1\mathrm{m^2}$；水平理想流体。
连续性 $S_1v_1=S_2v_2 \Rightarrow v_2=2v_1$
$$\Delta p=\frac12\rho(v_2^2-v_1^2)=\frac32\rho v_1^2$$
$$v_1=\sqrt{\frac{2\Delta p}{3\rho}}=\sqrt{\frac{2\times10^6}{3\times1000}}\approx25.82\mathrm{m/s}$$
$$Q=S_1 v_1=0.2\times25.82=\boldsymbol{5.16\ \mathrm{m^3/s}}$$



### 习题2.10

一根长水管直径为15 cm，其中充满水，水管的狭细部分直径为7.5 cm，如果在15 cm直径部分水流速为$1.2\ \mathrm{cm\cdot s^{-1}}$。(1)求狭细部分水的流速；(2)以$\mathrm{m^3\cdot s^{-1}}$为单位表示出水的流量。

🏍️解答：

$d_1=0.15\mathrm{m},\ d_2=0.075\mathrm{m},\ v_1=1.2\mathrm{cm/s}=0.012\mathrm{m/s}$
1. $S_1v_1=S_2v_2$，$v_2=v_1\left(\dfrac{d_1}{d_2}\right)^2=0.012\times 4=\boldsymbol{0.048\ \mathrm{m/s}}$
2. 流量
$$Q=S_1 v_1=\pi\left(\frac{0.15}{2}\right)^2\times0.012 \approx \boldsymbol{2.12\times 10^{-4}\ \mathrm{m^3/s}}$$



### 习题2.11

为了使救火水管中的水流可射达的竖直高度为20 m，问和水管相连接的总水管中需要的计示压强是多少？

🏍️解答：

水流竖直射高$H=20\mathrm{m}$；出口动能全部转化重力势能 $\dfrac12 \rho v^2=\rho g H$。
计\(p_{A示}=3\times10^5\mathrm{Pa}\)压强 $p=\dfrac12\rho v^2=\rho g H$
$$p=1000\times9.8\times20=\boldsymbol{1.96\times 10^5\ \mathrm{Pa}}$$



### 习题2.12

在水管的某一点$A$，水的流速为$1\ \mathrm{m\cdot s^{-1}}$，计示压强为$3\times10^5\ \mathrm{Pa}$。如果沿水管到另一点$B$，$B$点比$A$点高度降低了20 cm，$B$点水管的横截面积为$A$点的二分之一，求$B$点的计示压强。

🏍️解答：

已知条件

$v_A=1\ \mathrm{m/s}$，A点计压强 $p_{A示}=3\times 10^5\ \mathrm{Pa}$

B比A降低 $h_A-h_B=0.20\ \mathrm{m}$

$S_B=\frac12 S_A$，水密度 $\rho=1000\ \mathrm{kg/m^3}$

步骤1：连续性方程求$v_B$

$$S_A v_A = S_B v_B$$

$$v_B = v_A\cdot \frac{S_A}{S_B}=1\cdot \frac{S_A}{\frac12 S_A}= 2\ \mathrm{m/s}$$


步骤2：伯努利方程（计示压强，大气压$p_0$抵消）

$$p_{A示} \frac12\rho v_A^2+\rho g h_A = p_{B示} + \frac12\rho v_B^2+\rho g h_B$$
移项：
$$
p_{B示} = p_{A示} + \frac12\rho(v_A^2-v_B^2)+\rho g(h_A-h_B)
$$

代入数值：
$$
\begin{aligned}
\frac12\rho(v_A^2-v_B^2)&=0.5\times 1000\times(1^2-2^2)=500\times(-3)=-1500\ \mathrm{Pa}\\
\rho g(h_A-h_B)&=1000\times 9.8\times 0.20 = 1960\ \mathrm{Pa}
\end{aligned}
$$

$$
\begin{aligned}
p_{B示}&=3\times10^5 -1500 +1960 \\
&=300000 + 460 \\
&=3.0046\times 10^{5}\ \mathrm{Pa}
\end{aligned}
$$

### 习题2.13

在一横截面积为$10\ \mathrm{cm^2}$的水平管内有水流动，在管的另一段横截面积缩为$5\ \mathrm{cm^2}$。这两截面处的压强差为300 Pa。问一分钟内从管流出的水有多少立方米？

🏍️解答：

$S_1=10\mathrm{cm^2}=10\times10^{-4}\mathrm{m^2}$，$S_2=5\mathrm{cm^2}=5\times10^{-4}\mathrm{m^2}$，$\Delta p=300\mathrm{Pa}$。
连续性 $S_1v_1=S_2v_2 \Rightarrow v_2=2v_1$
$$\Delta p=\frac12\rho(v_2^2-v_1^2)=\frac32\rho v_1^2$$
$$v_1=\sqrt{\frac{2\Delta p}{3\rho}}=\sqrt{\frac{600}{3000}}=0.4472\ \mathrm{m/s}$$
$$Q=S_1 v_1=10\times10^{-4}\times 0.4472=4.472\times10^{-4}\ \mathrm{m^3/s}$$
一分钟流出体积
$$V=Q\cdot t=4.472\times10^{-4}\times60=\boldsymbol{0.0268\ \mathrm{m^3}}$$



### 习题2.14

从一水平管中排水的流量是$0.004\ \mathrm{m^3\cdot s^{-1}}$。管的横截面积为$0.01\ \mathrm{m^2}$处的绝对压强是$12\times10^5\ \mathrm{Pa}$。问管的横截面积压缩为多少时，才能使压强减少为$1.0\times10^5\ \mathrm{Pa}$？

🏍️解答：

$Q=0.004\mathrm{m^3/s}$；$S_1=0.01\mathrm{m^2},\ p_1=12\times10^5\mathrm{Pa}$；$p_2=1.0\times10^5\mathrm{Pa}$。
$v_1=\dfrac{Q}{S_1}=0.4\mathrm{m/s}$。
伯努利水平管：
$$p_1+\frac12\rho v_1^2 = p_2+\frac12\rho v_2^2$$
$$v_2=\sqrt{v_1^2+\frac{2(p_1-p_2)}{\rho}}=\sqrt{0.4^2+\frac{2\times11\times10^5}{1000}}\approx46.90\ \mathrm{m/s}$$
$$S_2=\frac{Q}{v_2}=\frac{0.004}{46.90}\approx \boldsymbol{8.53\times 10^{-5}\ \mathrm{m^2}}$$



### 习题2.15

一个顶部开口的圆筒容器，高为20 cm，直径为10 cm，在圆筒底部中心开一个横截面积为$1\ \mathrm{cm^2}$的小圆孔，水从圆筒顶部以$140\ \mathrm{cm^3\cdot s^{-1}}$的流量由水管注入圆筒，问圆筒中的水面可以升到多高？

🏍️解答：

注入流量$Q_\text{in}=140\mathrm{cm^3/s}$；小孔面积$S_2=1\mathrm{cm^2}$。平衡时流入=流出。托里拆利公式$v=\sqrt{2gh}$，$Q_\text{out}=S_2\sqrt{2gh}$。
$$Q_\text{in}=S_2\sqrt{2gh}$$
$$h=\frac{1}{2g}\left(\frac{Q_\text{in}}{S_2}\right)^2=\frac{1}{2\times980}\left(\frac{140}{1}\right)^2 \approx \boldsymbol{10\ \mathrm{cm}}$$



### 习题2.16

一个四壁竖直的大开口水槽，其中盛水，水深为$H$。在槽的一侧水面下$h$深处开一小孔。问：(1)射出的水流到地面时，距槽底边的距离是多少？(2)在槽壁上多高处再开一小孔，能使射出的水流有相同的射程？(3)要想得到最大射程，小孔要开在水面以下多深处？最大射程为多少？

🏍️解答：

水槽水深$H$，小孔水下深度$h$；小孔离地高度 $y=H-h$。托里拆利$v=\sqrt{2gh}$。

1\. 平抛运动：竖直 $y=\dfrac12 g t^2 \Rightarrow t=\sqrt{\dfrac{2(H-h)}{g}}$
水平射程
$$x=v\cdot t=\sqrt{2gh}\cdot\sqrt{\frac{2(H-h)}{g}}=\boldsymbol{2\sqrt{h(H-h)}}$$

2\. 相同射程：$h'$满足$h(H-h)=h'(H-h')$，解为$h'=H-h$；小孔距离水面深度为$\boldsymbol{H-h}$。

3\. 求$x=2\sqrt{h(H-h)}$最大值；$f(h)=h(H-h)$，$h=\dfrac H2$取最大。
小孔水下深度 $\boldsymbol{h=\dfrac H2}$；最大射程$\boldsymbol{x_\text{max}=H}$。



### 习题2.17

一圆筒中的水高为$H$，底面积为$S_1$，桶的底部有一面积为$S_2$的小孔。问桶中的水全部流尽需多长时间？

🏍️解答：

桶底小孔$S_2$，桶截面积$S_1$，初始高度$H$。

任意时刻高度$y$，流出$Q=S_2\sqrt{2gy}$；$-S_1 \mathrm{d}y = S_2\sqrt{2gy}\mathrm{d}t$。积分$y:H\to0$
$$T=\int_0^T \mathrm{d}t=\frac{S_1}{S_2}\int_0^H \frac{\mathrm{d}y}{\sqrt{2gy}}=\boldsymbol{\dfrac{S_1}{S_2}\sqrt{\dfrac{2H}{g}}}$$



### 习题2.18

贮有水的封闭大水箱，箱的上部引入气压为$8.0\times10^5\ \mathrm{Pa}$的压缩空气。箱的侧壁上距水面5.0 m处有一小孔，求水从此孔流出的速率。

🏍️解答：

水箱内部气压$p=8.0\times10^5\mathrm{Pa}$；小孔距水面$h=5.0\mathrm{m}$；出口大气压$p_0$。
伯努利，水面流速近似0：
$$p+\rho g h = p_0+\frac12\rho v^2$$
$$v=\sqrt{\frac{2(p-p_0)}{\rho}+2gh}$$
$p_0=1.013\times10^5\mathrm{Pa}$
$$v=\sqrt{\frac{2\times(8.0-1.013)\times10^5}{1000}+2\times9.8\times5}\approx \boldsymbol{38.2\ \mathrm{m/s}}$$



### 习题2.19

在皮托管中，用水作为压强计的液体，装在飞机上，用以测量空气的流速。如果水柱的最大高度差为0.1 m，问能测出空气的最大流速是多少？空气的密度是$1.3\ \mathrm{kg\cdot m^{-3}}$。

🏍️解答：

皮托管，水柱高差$\Delta h=0.1\mathrm{m}$；空气$\rho_\text{air}=1.3\mathrm{kg/m^3}$。
动压等于水柱压强：$\dfrac12\rho_\text{air} v^2=\rho_\text{水}g\Delta h$
$$v=\sqrt{\frac{2\rho_\text{水}g\Delta h}{\rho_\text{air}}}=\sqrt{\frac{2\times1000\times9.8\times0.1}{1.3}}\approx \boldsymbol{38.8\ \mathrm{m/s}}$$



### 习题2.20

自来水主管与文丘里咽喉管间的压强差为$10^5\ \mathrm{Pa}$，主管和细管的横截面积各为$0.1\ \mathrm{m^2}$和$0.05\ \mathrm{m^2}$。问管子中水的流量是多少？

🏍️解答：

$\Delta p=10^5\mathrm{Pa}$；$S_1=0.1\mathrm{m^2}$；$S_2=0.05\mathrm{m^2}$；$v_2=2v_1$。
$$\Delta p=\frac12\rho(v_2^2-v_1^2)=\frac32\rho v_1^2$$
$$v_1=\sqrt{\frac{2\Delta p}{3\rho}}=\sqrt{\frac{2\times10^5}{3000}}\approx 8.165\mathrm{m/s}$$
$$Q=S_1 v_1=0.1\times8.165=\boldsymbol{0.817\ \mathrm{m^3/s}}$$



### 习题2.21

有一水平放置的文丘里管，它的粗、细部分的直径分别为8 cm和4 cm；当水在管中流动时，连接在粗、细部分的竖直细管中水面的高度差为40 cm。计算水在粗、细部分的流速和流量。

🏍️解答：

文丘里管；$d_1=8\mathrm{cm},\ d_2=4\mathrm{cm}$；高差$\Delta h=40\mathrm{cm}=0.40\mathrm{m}$。
$S_1 v_1=S_2 v_2 \Rightarrow v_2=4v_1$；压强差$\Delta p=\rho g \Delta h$
$$\rho g\Delta h=\frac12\rho(v_2^2-v_1^2)=\frac12\rho(16v_1^2-v_1^2)=\frac{15}{2}\rho v_1^2$$
$$v_1=\sqrt{\frac{2g\Delta h}{15}}=\sqrt{\frac{2\times9.8\times0.4}{15}}\approx \boldsymbol{0.723\ \mathrm{m/s}}$$
$$v_2=4v_1=\boldsymbol{2.89\ \mathrm{m/s}}$$
$$S_1=\pi(0.04)^2\approx 5.027\times10^{-3}\mathrm{m^2}$$
$$Q=S_1 v_1 \approx \boldsymbol{3.63\times10^{-3}\ \mathrm{m^3/s}}$$



### 习题2.22

在$20^\circ \mathrm{C}$的空气中，一半径为$10^{-5}\ \mathrm{m}$、密度为$2\times10^3\ \mathrm{kg\cdot m^{-3}}$的球状灰尘微粒的沉积速度是多少？其在沉降时所受的阻力有多大？已知空气的黏度为$1.81\times10^{-5}\ \mathrm{Pa\cdot s}$。

🏍️解答：

$r=10^{-5}\mathrm{m},\ \rho=2\times10^3\mathrm{kg/m^3}$；空气$\eta=1.81\times10^{-5}\mathrm{Pa\cdot s}$。
终极沉降，重力=斯托克斯阻力：$\frac43\pi r^3(\rho-\rho_\text{air})g \approx 6\pi\eta r v$，忽略空气密度
$$v=\frac{2 r^2 \rho g}{9\eta}=\frac{2\times (10^{-5})^2\times 2000\times9.8}{9\times1.81\times10^{-5}} \approx \boldsymbol{2.41\times 10^{-2}\ \mathrm{m/s}}$$
阻力：$F=6\pi\eta r v \approx \boldsymbol{8.24\times10^{-11}\ \mathrm{N}}$



### 习题2.23

在液体中有一个空气泡，泡的直径为1 mm。设液体的黏度为$1.5\ \mathrm{Pa\cdot s}$，密度为$0.90\ \mathrm{g\cdot cm^{-3}}$。(1)求空气泡在该液体中上升的终极速度；(2)如果空气泡在水中上升，终极速度是多少？

🏍️解答：

气泡直径$1\mathrm{mm},\ r=5\times10^{-4}\mathrm{m}$；液体$\eta=1.5\mathrm{Pa\cdot s}$，$\rho_\text{liq}=900\mathrm{kg/m^3}$；气泡密度忽略。上升浮力等于阻力 $\dfrac43\pi r^3 \rho_\text{liq} g =6\pi\eta r v$
1. 该液体中
$$v=\frac{2 r^2 \rho_\text{liq} g}{9\eta}=\frac{2\times(5\times10^{-4})^2\times900\times9.8}{9\times1.5}\approx \boldsymbol{3.27\times 10^{-4}\ \mathrm{m/s}}$$
2. 在水中，$\rho_\text{liq}=1000\mathrm{kg/m^3}$
$$v=\frac{2\times(5\times10^{-4})^2\times1000\times9.8}{9\times1.5}\approx \boldsymbol{3.63\times 10^{-4}\ \mathrm{m/s}}$$



### 习题2.24

将一半径$R=1\ \mathrm{mm}$的钢球，轻轻放入装有甘油的缸中，当钢球的加速度是自由落体加速度的一半时，其速度是多少？钢球的最大速度是多少？已知钢球的密度$\rho_1=8.5\ \mathrm{g\cdot cm^{-3}}$，甘油的密度$\rho_2=1.32\ \mathrm{g\cdot cm^{-3}}$，甘油的黏度$\eta=8\ \mathrm{Pa\cdot s}$。

🏍️解答：

钢球$R=1\mathrm{mm}=10^{-3}\mathrm{m}$；$\rho_1=8500\mathrm{kg/m^3}$；甘油$\rho_2=1320\mathrm{kg/m^3},\ \eta=8\mathrm{Pa\cdot s}$。
受力：重力$G=\dfrac43\pi R^3\rho_1 g$；浮力$F_b=\dfrac43\pi R^3\rho_2 g$；阻力$f=6\pi\eta R v$。
牛顿第二定律：$G-F_b-f=ma$。

1\. $a=g/2$：
$$\frac43\pi R^3(\rho_1-\rho_2)g -6\pi\eta R v = \frac43\pi R^3\rho_1 \cdot \frac g2$$
化简得到
$$v=\frac{2R^2 g}{9\eta}\left(\rho_1-\rho_2-\frac{\rho_1}{2}\right)$$
代入：
$$v=\frac{2\times 10^{-6}\times9.8}{9\times8}\left(8500-1320-4250\right)\approx \boldsymbol{0.170\ \mathrm{m/s}}$$

2\. 终极速度$a=0$，$G-F_b=f$
$$v_T=\frac{2R^2 g(\rho_1-\rho_2)}{9\eta}=\frac{2\times10^{-6}\times9.8\times(8500-1320)}{9\times8}\approx \boldsymbol{1.95\ \mathrm{m/s}}$$

### 习题2.25

已知空气密度$\rho_{\text{空}}=1.25\ \mathrm{kg\cdot m^{-3}}$，黏度$\eta=1.81\times10^{-5}\ \mathrm{Pa\cdot s}$，雨滴密度$\rho_0=1\times10^3\ \mathrm{kg\cdot m^{-3}}$。一颗半径为0.06 mm的雨滴在空气中下落，假设其运动符合斯托克斯定律，求雨滴的终极速度及此速度下的雷诺数。

🏍️解答：

雨滴$r=0.06\mathrm{mm}=6\times10^{-5}\mathrm{m}$；$\rho_0=1000\mathrm{kg/m^3}$；空气$\rho_\text{空}=1.25\mathrm{kg/m^3}$，$\eta=1.81\times10^{-5}\mathrm{Pa\cdot s}$。
终极速度：
$$v_T=\frac{2r^2 g(\rho_0-\rho_\text{空})}{9\eta}\approx\frac{2\times(6\times10^{-5})^2\times9.8\times1000}{9\times1.81\times10^{-5}} \approx \boldsymbol{0.433\ \mathrm{m/s}}$$
雷诺数：
$$Re=\frac{\rho_\text{空}\,v_T \cdot 2r}{\eta}=\frac{1.25\times0.433\times 1.2\times10^{-4}}{1.81\times10^{-5}}\approx \boldsymbol{3.58}$$



### 习题2.26

牛奶在离心分离器中旋转，离心机转速$n=6\ 000$转每分，求在离转轴$x=5.0\times10^{-2}\ \mathrm{m}$处奶油油滴向转轴中心集中的终极速度。(牛奶可分离成奶油与奶液，奶油密度$\rho=0.94\times10^3\ \mathrm{kg\cdot m^{-3}}$，奶液密度$\rho'=1.03\times10^3\ \mathrm{kg\cdot m^{-3}}$，奶液黏度$\eta=1.1\times10^{-3}\ \mathrm{Pa\cdot s}$，奶油油滴直径$d=2.0\times10^{-6}\ \mathrm{m}$。)

🏍️解答：

离心机$n=6000\ \mathrm{r/min}$；$\omega=\dfrac{2\pi n}{60}=200\pi\ \mathrm{rad/s}$；$x=0.05\mathrm{m}$；奶油$\rho=940\mathrm{kg/m^3}$，奶液$\rho'=1030\mathrm{kg/m^3}$；油滴直径$d=2\times10^{-6}\mathrm{m},\ r=1\times10^{-6}\mathrm{m}$；$\eta=1.1\times10^{-3}\mathrm{Pa\cdot s}$。
离心场中等效惯性力：单位体积$(\rho-\rho')\omega^2 x$；终极速度，离心力=斯托克斯阻力
$$\frac43\pi r^3(\rho-\rho')\omega^2 x = 6\pi\eta r v$$
$$v=\frac{2 r^2 (\rho-\rho')\omega^2 x}{9\eta}$$
$(\rho-\rho')=-90\mathrm{kg/m^3}$，负号表\(p_{A示}=3\times10^5\mathrm{Pa}\)向转轴；取速率大小
$$|v|=\frac{2\times(10^{-6})^2\times90\times(200\pi)^2\times0.05}{9\times1.1\times10^{-3}}\approx \boldsymbol{4.01\times 10^{-4}\ \mathrm{m/s}}$$



### 习题2.27
水和甘油分别在两个直径均为0.1 m的管子中流动，流速均为$0.5\ \mathrm{m\cdot s^{-1}}$，求它们的雷诺数。已知室温时甘油的$\eta=830\times10^{-3}\ \mathrm{Pa\cdot s}$，$\rho=1.3\ \mathrm{g\cdot cm^{-3}}$。

🏍️解答：

管子直径$d=0.1\mathrm{m}$；流速$\bar v=0.5\mathrm{m/s}$。
水：$\rho_w=1000\mathrm{kg/m^3},\ \eta_w=1.005\times10^{-3}\mathrm{Pa\cdot s}$
$$Re_\text{w}=\frac{\rho_w \bar v d}{\eta_w}=\frac{1000\times0.5\times0.1}{1.005\times10^{-3}}\approx \boldsymbol{4.98\times10^4}$$

甘油：$\rho_g=1300\mathrm{kg/m^3},\ \eta_g=0.830\mathrm{Pa\cdot s}$
$$Re_\text{g}=\frac{1300\times0.5\times0.1}{0.830}\approx \boldsymbol{78.3}$$



### 习题2.28

$20^\circ \mathrm{C}$的水在半径为1.0 cm的管内流动。如果在管的中心处流速为$10\ \mathrm{cm\cdot s^{-1}}$，问由于黏性使得沿管长为2 m的两个截面间产生的压强差是多少？已知$20^\circ \mathrm{C}$时水的$\eta=1.005\times10^{-3}\ \mathrm{Pa\cdot s}$。

🏍️解答：

圆管层流，中心最大流速$v_\text{max}=0.10\mathrm{m/s}$；$R=0.01\mathrm{m}$；管长$L=2\mathrm{m}$；$\eta=1.005\times10^{-3}\mathrm{Pa\cdot s}$。
层流 $v_\text{max}=\dfrac{\Delta p R^2}{4\eta L}$
$$\Delta p=\frac{4\eta L v_\text{max}}{R^2}=\frac{4\times1.005\times10^{-3}\times2\times0.1}{0.01^2}=\boldsymbol{80.4\ \mathrm{Pa}}$$



### 习题2.29

温度为$37^\circ \mathrm{C}$时，水的黏度为$6.91\times10^{-4}\ \mathrm{Pa\cdot s}$，水在长度为0.2 m，半径为$1.5\times10^{-3}\ \mathrm{m}$的水平管内流动，当管子两端压强差为$\Delta p=4\times10^3\ \mathrm{Pa}$时，求水的流量。

🏍️解答：

泊肃叶公式 $Q=\dfrac{\pi R^4 \Delta p}{8\eta L}$；
$R=1.5\times10^{-3}\mathrm{m},\ L=0.2\mathrm{m},\ \Delta p=4\times10^3\mathrm{Pa},\ \eta=6.91\times10^{-4}\mathrm{Pa\cdot s}$
$$Q=\frac{\pi\times(1.5\times10^{-3})^4\times 4\times10^3}{8\times6.91\times10^{-4}\times0.2}\approx \boldsymbol{9.10\times 10^{-5}\ \mathrm{m^3/s}}$$



### 习题2.30

在一个0.25 m深的大甘油槽中盛有甘油，甘油的密度$\rho=1.32\times10^3\ \mathrm{kg\cdot m^{-3}}$，黏度为$\eta=0.83\ \mathrm{Pa\cdot s}$。槽底接一长为0.25 m，内径为$3\times10^{-3}\ \mathrm{m}$的竖直管。问稳定流动时竖直管管心处流速是多少？

🏍️解答：

竖直圆管，总驱动压强来自甘油液柱$\Delta p=\rho g H$；$H=0.25\mathrm{m}$；管长$L=0.25\mathrm{m}$；$R=1.5\times10^{-3}\mathrm{m}$，$a=0.83\mathrm{Pa\cdot s}$，$\rho=1320\mathrm{kg/m^3}$。
层流轴心最大流速
$$v_\text{max}=\frac{\Delta p R^2}{4\eta L}=\frac{\rho g H R^2}{4\eta L}$$
$H=L$，化简
$$v_\text{max}=\frac{\rho g R^2}{4\eta}=\frac{1320\times9.8\times(1.5\times10^{-3})^2}{4\times0.83}\approx \boldsymbol{8.76\times 10^{-3}\ \mathrm{m/s}}$$



### 习题2.31

水通过内径为0.20 m的管子，从水塔底部流出，水塔内水面高出排水管出口25 m。如果维持水位差不变，并设每立方米水通过全部管路能量损失为$2.40\times10^5\ \mathrm{J}$，试问每小时由管子排出的水量为多少立方米？

🏍️解答：

水塔水位$H=25\mathrm{m}$；每立方米能量损失$w=2.40\times10^5\mathrm{J/m^3}$；$d=0.20\mathrm{m}$。
单位体积总势能$\rho g H$，一部分用于出口动能，一部分损失。
$$\rho g H = w+\frac12\rho v^2$$
$$\frac12\rho v^2=\rho g H-w =1000\times9.8\times25-2.40\times10^5=5000\ \mathrm{J/m^3}$$
$$v=\sqrt{\frac{2\times5000}{1000}}=\sqrt{10}\approx3.162\mathrm{m/s}$$
截面积$S=\pi(0.1)^2=0.031416\mathrm{m^2}$
$$Q=S\cdot v=0.031416\times3.162\approx0.0993\mathrm{m^3/s}$$
每小时体积
$$V=Q\times3600\approx \boldsymbol{357\ \mathrm{m^3/h}}$$



### 习题2.32

血液流过一根长为1 mm、半径为2 mm的毛细血管，若流过毛细血管中心血流速率为$0.66\ \mathrm{mm\cdot s^{-1}}$，求此段毛细血管的血压降。已知血液黏度$\eta=4\times10^{-3}\ \mathrm{Pa\cdot s}$。

🏍️解答：

层流，中心流速$v_\text{max}=0.66\mathrm{mm/s}=6.6\times10^{-4}\mathrm{m/s}$；$R=2\mathrm{mm}=2\times10^{-3}\mathrm{m}$；$L=1\mathrm{mm}=1\times10^{-3}\mathrm{m}$，$\eta=4\times10^{-3}\mathrm{Pa\cdot s}$。
$$v_\text{max}=\frac{\Delta p R^2}{4\eta L}$$
$$\Delta p=\frac{4\eta L v_\text{max}}{R^2}=\frac{4\times4\times10^{-3}\times 1\times10^{-3}\times 6.6\times10^{-4}}{(2\times10^{-3})^2}=\boldsymbol{2.64\times 10^{-3}\ \mathrm{Pa}}$$



### 习题2.33

人的心脏不断向大动脉泵送血液，进入大动脉的最大流量约为$500\ \mathrm{cm^3\cdot s^{-1}}$。假设大动脉的直径为2.5 cm，流动大致可认为是层流，血液是牛顿流体。求：(1)大动脉内单位长度上的压强差；(2)轴心处的流动速度。已知血液黏度$\eta=4\times10^{-3}\ \mathrm{Pa\cdot s}$。

🏍️解答：

$Q=500\mathrm{cm^3/s}=5\times10^{-4}\mathrm{m^3/s}$；$D=2.5\mathrm{cm}$，$R=0.0125\mathrm{m}$；$\eta=4\times10^{-3}\mathrm{Pa\cdot s}$。
泊肃叶 $Q=\dfrac{\pi R^4}{8\eta}\cdot \dfrac{\Delta p}{L}$
1. 单位长度压强差 $\dfrac{\Delta p}{L}$
$$\frac{\Delta p}{L}=\frac{8\eta Q}{\pi R^4}=\frac{8\times4\times10^{-3}\times5\times10^{-4}}{\pi\times(0.0125)^4}\approx \boldsymbol{167\ \mathrm{Pa/m}}$$

2. 层流平均流速 $\bar v=\dfrac{Q}{\pi R^2}$；轴心最大流速 $v_\text{max}=2\bar v$
$$\bar v=\frac{5\times10^{-4}}{\pi\times0.0125^2}\approx 1.019\mathrm{m/s}$$
$$v_\text{max}=2\bar v=\boldsymbol{2.04\ \mathrm{m/s}}$$



### 习题2.34

在其他参量不变的条件下，要使血管的流体力学阻尼(用$\Delta p/Q$表示)增大30%，半径需要改变多少？假设该情况为牛顿流体的层流行为。

🏍️解答：

流体阻尼 $Z=\dfrac{\Delta p}{Q}$；泊肃叶 $Q=\dfrac{\pi R^4 \Delta p}{8\eta L}$，$Z=\dfrac{8\eta L}{\pi R^4}$。
$Z \propto R^{-4}$；$Z_2=1.3 Z_1$。
$$\frac{Z_2}{Z_1}=\left(\frac{R_1}{R_2}\right)^4=1.3$$
$$\frac{R_2}{R_1}=1.3^{-1/4}\approx 0.933$$
半径变为原来的0.933倍；**半径减小约6.7%**。